install/kubernetes/helm/deployIstioHelmCharts.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+echo "Deploying Istio..."
+
+echo "Creating RBAC rules for Istio..."
+
+helm install istio-rbac --name istio-rbac
+retCode=$?
+if [ $retCode != '0' ]; then
+  echo "Creating of RBAC rules failed, exiting..."
+  exit 1 
```

Comment:
Knowing the exit code from helm is more important than knowing the step in which the Helm deployment failed.  That is indicated by the echo operation, however, the helm return code is never indicated.  Please change exit 1 to exit $retCode.

Reply:
done